pt-BR/action_view_overview.md

```diff
@@ -4,22 +4,22 @@
 Action View Overview
 ====================
 
-After reading this guide, you will know:
+Depois de ler este guia, você vai saber:
 
-* What Action View is and how to use it with Rails.
-* How best to use templates, partials, and layouts.
-* How to use localized views.
+* O que é *Action View* e como é utilizada no Rails.
+* A melhor forma de usar *templates*, *partials* e *layouts*.
+* Como utilizar *views* localizadas.
 
 --------------------------------------------------------------------------------
 
-What is Action View?
---------------------
+O que é *Action View*?
+----------------------
 
-In Rails, web requests are handled by [Action Controller](action_controller_overview.html) and Action View. Typically, Action Controller is concerned with communicating with the database and performing CRUD actions where necessary. Action View is then responsible for compiling the response.
+No *Rails*, as requisições web são tratadas por [*Action Controller*](action_controller_overview.html) e *Action View*. Normalmente, o *Action Controller* é responsável por se comunicar com o banco de dados e realizar ações de *CRUD* quando necessário. A *Action View* é responsável por compilar a resposta.
 
-Action View templates are written using embedded Ruby in tags mingled with HTML. To avoid cluttering the templates with boilerplate code, a number of helper classes provide common behavior for forms, dates, and strings. It's also easy to add new helpers to your application as it evolves.
+Os templates *Action View* são escritos usando Ruby embutido em tags mescladas com HTML. Para evitar poluir os templates com código clichê (*boilerplate code*), uma variedade de classes utilitárias (*helpers*) disponibilizam comportamentos comuns para lidar com forms, datas e strings. Também é fácil adicionar novas classes utilitárias (*helpers*) em sua aplicação conforme ela evolui.
```

Comment:
```suggestion
Os templates *Action View* são escritos usando Ruby embutido em tags mescladas com HTML. Para evitar poluir os templates com código clichê (*boilerplate code*), uma variedade de classes utilitárias (*helpers*) disponibilizam comportamentos comuns para lidar com *forms*, datas e *strings*. Também é fácil adicionar novas classes utilitárias (*helpers*) em sua aplicação conforme ela evolui.
```

Aqui acho que podemos colocar itálico em alguns outros termos, você acha valido? Nada que doa também